src/wolfkrow/core/engine/task_graph.py

```diff
@@ -271,11 +255,11 @@ def execute_local(
             stdout, stderr = process.communicate()
 
             if process.returncode == 0:
-                logging.info("Task '%s' Successfully completed" % task['obj'].name)
-                results[task['obj'].name] = True
+                logging.info("Task '%s' Successfully completed" % task['obj'].task_name)
+                results[task['obj'].task_name] = True
             else:
-                logging.error("Task '%s' Failed. Will skip all dependant tasks." % task['obj'].name)
-                results[task['obj'].name] = False
+                logging.error("Task '%s' Failed. Will skip all dependant tasks." % task['obj'].task_name)
+                results[task['obj'].task_name] = False
```

Comment:
`task` here is no longer a dict. This needs to be updated to use the new TaskExport class